app/src/main/java/eu/depau/etchdroid/job/JobProcedureBuilder.kt

```diff
@@ -24,14 +25,16 @@ class JobProcedureBuilder() : IJobProcedureBuilder, KotletParcelable {
 
 
     override fun build(context: Context): IJobProcedure {
-        assert(::jobType.isInitialized) { "Job type not set, aborting" }
```

Comment:
`Assertions are never enabled in Android.`

Reply:
But they are in tests, put back.

Reply:
@Depau 

> so that assert works only when you are testing the app, but on release versions assert will not be called
> BuildConfig.DEBUG will be false when you export a release build.

[Stackoverflow](https://stackoverflow.com/questions/23113497/what-are-buildconfig-debug-conditional-checks)

Reply:
That's exactly the intended behavior, they should trigger only in unit tests.